qemu/tests/hpt_miscellaneous.py

```diff
@@ -88,6 +103,16 @@ def run(test, params, env):
         # For HPT reszing after hotplug memory
         hpt_mem = MemoryHotplugTest(test, params, env)
         hpt_mem.hotplug_memory(vm, "hpt_mem")
+        orig_mem = int(session.cmd_output(cmd=params['free_mem_cmd']))
+        in_mem = int(
+            utils_numeric.normalize_data_size(params['size_plug'], 'B'))
+        logging.debug('Guest free memory size is %d bytes' % orig_mem)
+        plug_timeout = float(params.get('plug_timeout', 20))
+        if not utils_misc.wait_for(
+                 lambda: check_mem_increase(session, params, orig_mem, in_mem),
+                 plug_timeout):
+            test.fail("Guest memory size is not increased %s in %s sec."
+                      % (params['size_plug'], params.get('plug_timeout', 20)))
```

Comment:
`% (params['size_plug'], params.get('plug_timeout', 20)))` -> `% (params['size_plug'], plug_timeout)`

Reply:
Updated it and thanks